lib/cron.js

```diff
@@ -394,7 +394,13 @@
 
 	function CronJob(cronTime, onTick, onComplete, startNow, timeZone, context, runOnInit, utcOffset) {
 		var _cronTime = cronTime;
-		if (typeof cronTime != 'string' && arguments.length == 1) {
```

Comment:
How is this incorrect and how does your code fix it without adding unnecessary processing?